src/Umbraco.Web.UI.Client/src/views/propertyeditors/colorpicker/colorpicker.html

```diff
@@ -5,10 +5,11 @@
         <localize key="colorpicker_noColors">You haven't defined any colors</localize>
     </div>
 
-    <umb-color-swatches colors="model.config.items"
-                        selected-color="model.value.value"
-                        size="m"
-                        use-label="model.useLabel">
+    <umb-color-swatches 
+        colors="model.config.items"
+        selected-color="model.value"
+        size="m"
+        use-label="model.useLabel">
```

Comment:
@madsrasmussen not sure if something specific has changed on this in v8, but when I added this, the `selected-color` was just the hex color stored in `model.value´ as an object `{ "label": "Test", "value": "ff0000" }`
https://github.com/umbraco/Umbraco-CMS/blob/1bb593d264a085f94a3ce3bd710392b350561430/src/Umbraco.Web.UI.Client/src/views/propertyeditors/colorpicker/colorpicker.controller.js#L30

Does this change still set the `active` class?
https://github.com/umbraco/Umbraco-CMS/blob/dd6e764588d22ef2b7bce01fd504ece89834f181/src/Umbraco.Web.UI.Client/src/views/components/umb-color-swatches.html#L3